source/reference/methods.txt

```diff
@@ -876,3 +876,99 @@ Role Management Methods
    * - :method:`db.updateRole()`
 
      - Updates a user-defined role.
+
+Replication Methods 
+-------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+ 
+   * - ``rs.printSecondaryReplicationInfo``
+
+     - Returns the status of the secondary members of the replica set. 
+     
+       This is identical to the ``db.printSecondaryReplicationInfo()`` 
+       method. This method returns the same output as the 
+       :method:`rs.printSlaveReplicationInfo()` method in the legacy 
+       :binary:`~mongo` shell. The :method:`rs.printSlaveReplicationInfo()` method is deprecated in ``mongosh``. Use 
```

Comment:
[nit] For the second instance of :method:`rs.printSlaveReplicationInfo()`, suggest just using monospace instead of the `method` link. I don't think it's super helpful to provide two links to the same location immediately after one another.

Reply:
I would also be wary of saying it 'return the same output' since it's formatted differently.